docs/using-bi-for-auth.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Using Beyond Identity for Authentication
+sidebar_position: 2
+---
+In this guide we will show you how to create a tenant, an application, configure the application, and how to integrate with the Beyond Identity Embedded SDK
+
+### Tenant Setup
+
+Create a tenant via the self signup form:
+
+https://console-us.beyondidentity.com/signup
+
+After creating a tenant, you will be redirected to an Admin Console for the newly created tenant.
+
+### Create a new Realm
+
+Click the drop down on the top left corner and add a new realm. [[Screenshot](./screenshots/NewRealm.jpg)] 
```

Comment:
I would add this above for context (from the other doc):

"Realms are unique administrative domains within a tenant. All new tenants have a default realm called Beyond Identity Admin which should not be used to configure for delegate IDP purposes."